packages/js/wasm/src/WasmWrapper.ts

```diff
@@ -60,6 +62,43 @@ export class WasmWrapper implements Wrapper {
     Tracer.endSpan();
   }
 
+  static async from(
+    manifestBuffer: Uint8Array,
+    wasmModule: Uint8Array,
+    options?: GetManifestOptions
+  ): Promise<WasmWrapper>;
+  static async from(
+    manifestBuffer: Uint8Array,
+    wasmModule: Uint8Array,
+    fileReader: IFileReader,
+    options?: GetManifestOptions
+  ): Promise<WasmWrapper>;
+  static async from(
+    manifestBuffer: Uint8Array,
+    fileReader: IFileReader,
+    options?: GetManifestOptions
+  ): Promise<WasmWrapper>;
+  static async from(
+    fileReader: IFileReader,
+    options?: GetManifestOptions
+  ): Promise<WasmWrapper>;
+  static async from(
+    manifestBufferOrFileReader: Uint8Array | IFileReader,
+    wasmModuleOrFileReaderOrManifestOptions?:
+      | Uint8Array
+      | IFileReader
+      | GetManifestOptions,
+    fileReaderOrManifestOptions?: IFileReader | GetManifestOptions,
+    manifestOptions?: GetManifestOptions
+  ): Promise<WasmWrapper> {
```

Comment:
Should this return a Result?

Reply:
Yah I think we should start propagating errors instead of throwing.

Reply:
This method is intended to be used (and be very ergonomic) when the developer knows it will not fail (or doesn't care) because he's using compiled wrapper binaries (manifest and wasm module buffers).
E.g.
```typescript
const client = new PolywrapClient({
  resolver: RecursiveResolver.from([
     ...,
     { uri: "some-uri", wrapper: WasmWrapper.from(manifestBuffer, wamModule) }
  ])
});
```

If the developer wants a `safe` version (that returs Result) he can use WasmPackage.from().createWrapper() which is less ergonomic but gets the job done :)